app/assets/javascripts/autocomplete_checkboxes.js

```diff
@@ -0,0 +1,90 @@
+function hitEnter(event, element, formType, inputType) {
+  if (event.which === 13) {
+    enterOnAutocomplete(element, formType, inputType);
+    return false;
+  }
+}
+
+function openQuickCreate(input, inputType) {
+  showBackdrop();
+  $("#" + inputType + "_quick_create").toggleClass("display_none");
```

Comment:
That's a good point, hmm sounds like we can just refactor that all over the codebase XD